spec/service_consumers/provider_states.rb

```diff
@@ -1,5 +1,15 @@
 # frozen_string_literal: true
 
+Pact.provider_states_for 'Forms' do
+  provider_state 'single form exists' do
+    set_up do
+      return if Rails.env.production?
```

Comment:
what's the scenario where this is run in production?

Reply:
I would hope never, ever. But in the unlikely case that was tried by accident, I want to ensure we don't create garbage in the production database.

Reply:
I had a hunch this was the case.  Is there some other level this could hook in?  Does the `pact` gem not do this by default?  This makes it seem like it would be needed for each pact set_up we do